packages/ui/src/theme/css/styles.scss

```diff
@@ -20,6 +20,23 @@ html:focus-within {
   scroll-behavior: smooth;
 }
 
+// Remove default margin
+// https://piccalil.li/blog/a-modern-css-reset/
+body,
```

Comment:
Can we scope this to body only since that's the root of the issue? I don't see a reason to reset the other elements at this time and it may mess up the styling of our other primitives.

Reply:
I made this change and decided to remove the `<h1>` tag from the next examples since it isn't necessary and the other examples don't have a similar tag.